nix/modules/fedimintd.nix

```diff
@@ -293,7 +325,10 @@ in
             RestrictAddressFamilies = [
               "AF_INET"
               "AF_INET6"
+              "AF_UNIX"
+              "AF_NETLINK"
```

Comment:
Adding these two finally made it not panic in iroh mode with `IO Address family not supported by protocol (os error 97)`. I still need to figure out if both are needed.

Reply:
`AF_NETLINK`was the one required

Reply:
I opened an issue on iroh out of curiosity: https://github.com/n0-computer/iroh/issues/3265